src/proxy.js

```diff
@@ -185,6 +185,7 @@ const mapTraps = {
     }
 }
 const mapGetters = {
+    [DRAFT_STATE]: state => state,
```

Comment:
Ah, thanks for catching this!

Reply:
Np. Thanks for refactoring the whole thing!